pkgs/development/python-modules/color-operations/default.nix

```diff
@@ -42,6 +42,11 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "color_operations" ];
 
+  disabledTests = lib.optionals stdenv.isAarch64 [
+    # AsserionError: assert -2.220446049250313e-16 >= 0
+    "test_rgb2lch"
```

Comment:
Would you be OK to [report this upstream](https://github.com/vincentsarago/color-operations/issues/new)?

Reply:
Sure ! Done: https://github.com/vincentsarago/color-operations/issues/9

Reply:
The test has been fixed upstream.